HelpMyStreetFE/HelpMyStreetFE/src/sass/components/awards/awards.scss

```diff
@@ -0,0 +1,88 @@
+﻿
+.user-badge{
+    display: flex;
+
+    @media screen and (max-width: $sm-screen){
+        align-self: center;
+    }
+}
+.user-detail{
+    display: inline-flex;
+    align-items: flex-start;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.tooltiptext {
+    visibility: hidden;
```

Comment:
Tooltip positioning could do with tweaking a little - as an unverified user it's hiding the "Verify your ID" link

![image](https://user-images.githubusercontent.com/58040070/94671895-61c93800-030c-11eb-90ef-396ac0f2e0c9.png)